lib/ex_prompt.ex

```diff
@@ -17,29 +17,42 @@ defmodule ExPrompt do
   @doc """
   Reads a line from `:stdio` displaying the prompt that is passed in.
 
-  In case of any errors or `:eof` this function will return and empty string.
+  In case of any errors or `:eof` this function will return the default value.
 
   ## Examples
 
   To ask a user for their name and await for the input:
 
     ExPrompt.string("What is your name?\n")
+
+  To ask for a hostname and if none is informed, defaults to `localhost`:
+
+    ExPrompt.string("What is the hostname?\n", "localhost")
   """
   @spec string(prompt) :: String.t()
-  def string(prompt) do
-    case IO.gets(prompt) do
-      :eof -> ""
-      {:error, _reason} -> ""
-      str -> String.trim_trailing(str)
+  @spec string(prompt, String.t()) :: String.t()
+  def string(prompt, default \\ "") do
+    case IO.gets(string_prompt_formatter(prompt, default)) do
+      :eof ->
+        ""
+
```

Comment:
Let's remove the spaces between the branches

Reply:
Mix format fault... :-(
I'll remove the spaces.